testsuite/tests/templates/basic/float64_monoid_utils_2.ml

```diff
@@ -0,0 +1,10 @@
+let one = Float64_monoid.empty
```

Comment:
I think that in an ideal world, `float64_monoid_utils_2` should get a name that explains how it is interestingly different from `float64_monoid_utils`, instead of just a `_2`.  Maybe `float64_monoid_utils_mixed`?  (or whatever name you think is best)